.github/workflows/build-docs.yaml

```diff
@@ -2,28 +2,16 @@ name: build docs
 
 on:
   push:
-    branches:
-      - '**'
```

Comment:
Auch hier müsste es wieder aktiv sein für das Required

Reply:
Alle Branches ist meiner Meinung nach der Default. Siehe https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore:~:text=If%20you%20define%20neither%20tags/tags%2Dignore%20or%20branches/branches%2Dignore%2C%20the%20workflow%20will%20run%20for%20events%20affecting%20either%20branches%20or%20tags

Reply:
https://github.com/it-at-m/digiwf-core/pull/1443/files#r1524355012